### PR TITLE
Fix deprecations

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
         "kriswallsmith/buzz": "~0.15"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~4.0",
+        "symfony/phpunit-bridge": "^2.7.4"
```
Comment (cedric-g) [inline composer.json]: PR #59 and this one are not consistent, in #59 you remove the PHPUnit dependency and in this one you keep it and add the symfony's phpunit-bridge.

Comment (Soullivaneuh) [inline composer.json]: I don't keep it. It's just PR separation.

I'll rebase it once when #59 will be merged.

**Comment by Soullivaneuh** [comment]
@cedric-g PR rebased. Could you please push a patch release after this one?

**Comment by Soullivaneuh** [comment]
Thanks for merge and patch. :+1:

**Pull request merged**